datamodels/attestation-results/grammar/json-labels.cddl

```diff
@@ -1,15 +1,18 @@
 ; $ar-status
-ar.status-failure = "failure"
-ar.status-success = "success"
-ar.status-unknown = "unknown"
+ar.trust-tier-none = "none"
+ar.trust-tier-affirming = "affirming"
+ar.trust-tier-warning = "warning"
+ar.trust-tier-contraindicated = "contraindicated"
 
 ; attestation-results
-ar.status = "status"
```

Comment:
if this label is removed the field becomes anonymous.

Reply:
argh, yes, that's my bad.

Reply:
awesome, thanks!